frontend/src/container/MetricsApplication/Tabs/Overview/ApDex/ApDexMetrics.tsx

```diff
@@ -84,7 +84,7 @@ function ApDexMetrics({
 	return (
 		<Graph
 			name="apdex"
-			filterNaN
+			fillSpans={false}
```

Comment:
any reason to make fillSpans false here ?

Reply:
`fillSpans = true` sets the values to 0, when `fillSpans = false` `NaN` & No Values for the timestamp are set to `null`